src/assets/scripts/base/PositionModel.js

```diff
@@ -267,49 +246,26 @@ export default class PositionModel {
  */
 PositionModel.calculatePosition = (coordinates, referencePostion, magneticNorth) => {
     if (!coordinates || !referencePostion || !magneticNorth) {
-        throw new TypeError('Invalid parameter. PositionModel.getPosition() requires coordinates, referencePostion and magneticNorth as parameters');
+        throw new TypeError('Invalid parameter. PositionModel.getPosition() requires coordinates, referencePostion ' +
+            'and magneticNorth as parameters');
```

Comment:
We talked about this briefly over slack.  When it comes to error messages, I'm ok with exceeding the min line length, as long as they aren't too long, and adding an `// eslint-ignore-line max-len` comment.  

You are correct to break lines in this case, however. 👍